recipes-core/images/avnet-image-minimal.inc

```diff
@@ -51,6 +51,12 @@ IMAGE_INSTALL:append:zynqmp = "\
         util-linux-mkfs \
 "
 
+IMAGE_INSTALL:append:mz7020-iocc = "\
```

Comment:
Will these additional packages only apply to mz7020-iocc and not the mz7010-iocc as well?